### PR TITLE
Add Notifications screen for unified prologue carousel

### PR DESCRIPTION
**Pull request by frosty**
Ref #15056. This PR adds the Notifications screen to the prologue carousel.

| iPhone | iPad |
|---|---|
| Simulator Screen Shot - iPhone 12 Pro - 2021-03-29 at 10 37 37 | Simulator Screen Shot - iPad Pro (9 7-inch) - 2021-03-29 at 10 41 30 |

**To test**

* Build, run and log out if necessary
* Verify that the notifications (third) page of the login prologue is consistent with the designs (see issue #15056; see also example screenshot below)

PR submission checklist:

- [x] I have considered adding unit tests where possible.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/WordPress.xcodeproj/project.pbxproj
Comment (Gio2018) [inline WordPress/WordPress.xcodeproj/project.pbxproj:8741]: we should probably move this file in a separate group or, if we feel that we have too many sub-groups, just remove the topic level (e.g. `Editor`).

**Review by Gio2018 — CHANGES_REQUESTED** [review]
Hey @frosty , as discussed on Slack, there's a crash happening, likely due to `NSAttributedString` not being called from the main thread. Other than that, I left a code comment.

**Comment by frosty** [comment]
@Gio2018 Thanks for catching this! It turns out that the NSAttributedString construction from HTML needs to be run on the main thread, but there's no guarantee that SwiftUI will call it there. I've gone for a simpler approach to avoid having to use this at all. Let me know what you think.

File: WordPress/WordPress.xcodeproj/project.pbxproj
Comment (frosty) [reply WordPress/WordPress.xcodeproj/project.pbxproj:8741]: Good catch – I'll wait until this is merged to address this as I don't want to distrupt any other PRs.

**Review by Gio2018 — APPROVED** [review]
Works great now!

A few minor comments comparing to the designs:

- the "reply" icon should be bigger and moved a bit to the right
- the "star" icon maybe moved just a bit lower and a bit to the right
- the "comments" icon maybe moved a bit to the left

All these are of course super minor and can be safely ignored.

**Pull request merged**
